scripts/format-queries.lua

```diff
@@ -0,0 +1,286 @@
+-- #!/usr/bin/env -S nvim -l
```

Comment:
```suggestion
#!/usr/bin/env -S nvim -l
```
Is it intentional that you keep reverting this change (and the `+x`)?

Reply:
No, I forgot to apply it

Reply:
Just wondering.